### PR TITLE
Create the rails project

### LINKED ISSUE #3: Create the rails project
# REQUIREMENTS
Create the rails project 
- Configure database connection with PostgreSQL

# DESIGN

### PR DESCRIPTION
**Pull request by DiegoSteve**
# Description
What did you implemented/Why did you implemented this:
 -What?: Create a initialize rails project
-Why?: Was created to set all the initial configuration to work with Rails

 # Testing
 Help me how can I test or look at the changes

  ## Screenshots
Captura de Pantalla 2022-03-08 a la(s) 17 55 04

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+#     url: <%= ENV["MY_APP_DATABASE_URL"] %>
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
+production:
+  <<: *default
+  database: kata_07_reloj_checador_k7_f22_t4_production
+  username: kata_07_reloj_checador_k7_f22_t4
+  password: <%= ENV["KATA_07_RELOJ_CHECADOR_K7_F22_T4_DATABASE_PASSWORD"] %>
```
Comment (JuanequeX) [inline config/database.yml]: This file has your project's env-vars and should be in your .gitignore file
should be replaced with the file database.example.yml to have references to which username and database to use when someone downloads your project.

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 1 above)
Comment (YilKanda) [reply config/database.yml]: I agree with @JuanequeX

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 1 above)
Comment (Frcomparan) [reply config/database.yml]: Thanks for the review, we did the suggested changes

**Review by YilKanda — CHANGES_REQUESTED** [review]
Remember, you must leave the database.yml.example file so that the user who downloads your project knows that there must be a file called database.yml

**Comment by brocha05** [comment]
We already add a file named database.example.yml in config directory, there we add an example of the neccesary values for the correct function of the database.

**Review by HeikelV — APPROVED** [review]
LGTM

**Review by brocha05 — APPROVED** [review]
(Y)

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]

**Pull request merged**